scripts/devcontainer/_wait-until-server-ready

```diff
@@ -3,8 +3,22 @@
 
 set -euo pipefail
 
+base_url="$DARK_CONFIG_STATIC_HOST"
+
+for cmd in "$@"; do
+  case $cmd in
+    --use-apiserver )
```

Comment:
I think I would rather see this split out into a script called `wait-until-apiserver-ready`